io.edcd.EDMarketConnector.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
We can't include a static desktop file upstream, because there's no telling what other ways users might be installing and running the application under Linux, so paths can easily differ.

Reply:
except that you shouldn't set a path. Lack of .desktop for a desktop app is a bug. It's a 20+ year old standard.

Reply:
On Linux the application is "run the main python file".  So either you need to specify the path to that within the .desktop file, or would need to point that at a shell script which ... would need to specify the path to the main python file.

This isn't a python application with an egg or wheel, you just run it from the source, so that could be located anywhere the user chose to clone/unpack it.

In the case of this flatpak the location is known, so *it* can have a working .desktop file.

Reply:
I'm with @Athanasius here. You can also see it differently - it's an application built for and focused on Windows and we are just happy that you can also run it on Linux. I get your point, but it's not a fixed rule and other applications [have it as well](https://github.com/flathub/com.visualstudio.code/blob/master/com.visualstudio.code.desktop).

Reply:
We have an upstream PR https://github.com/EDCD/EDMarketConnector/pull/1800 that should address this.